manifests/server/default_privileges.pp

```diff
@@ -140,6 +141,21 @@
       }
       $_check_type = 'T'
     }
+    'SCHEMAS': {
+      if (versioncmp($version, '10') == -1) {
```

Comment:
Perhaps this works better?
```suggestion
      unless (versioncmp($version, '10') >= 0) {
```

Reply:
I think this would be quite surprising since it would silently ignore attempts to alter schema privileges on unsupported platforms, rather than warning about the situation? Probably the result is that the desired database security is not there and it's either inaccessible or insecure...

Reply:
I think what I suggested actually makes the check work, but tests should properly indicate this.

Reply:
Ah I misread very badly! I don't know what the preferred semantics is ("fail if X" vs "fail unless not X") so I can invert that if desired. BTW just in case you saw the older comments and not the most recent one, it's perhaps worth repeating that the tests that didn't time out look successful now (I changed `10.0` to `10`, having not realised this behaviour of `versioncmp`)

Reply:
Yes, there's some odd behavior and I think this syntax will work with both `9.x` and `10`, `11` etc.

Reply:
so the version supplied is `'10'`, `versioncmp('10', '10.0') == -1` and `versioncmp('10', '10') == 0`, so the check was triggering incorrectly and is no longer. Also `versioncmp('10.0', '10') == 1`, so even if a longer version string were passed it should still be OK.

I am not sure if you still want me to switch out `if` for `unless` and invert the check though?

Reply:
If this implementation works then I'm fine with it and no need to change it.

Reply:
OK, let's keep it that way - as far as I can tell the only issue now is the timed out CI runs which I believe are ever-present!